### ISSUE TITLE
functional tests broken by any Jinja2 in global-tests.cylc

### ISSUE THREAD

**Issue by hjoliver** [cylc/cylc-flow#5022]
`create_test_global_config` shell function needs tweaking

**Comment by wxtim** [comment]
Replicated. 😢

**Comment by MetRonnie** [comment]
Is this about trying to do something like this?
```shell
create_test_global_config "" "
#!Jinja2
[scheduler]
    process pool timeout = {{ X }}
"
```
If so I think the simple solution is?
```shell
create_test_global_config "#!Jinja2" "
[scheduler]
    process pool timeout = {{ X }}
"
```

The two args to `create_test_global_config` are pre- and ap-pended to the global config respectively

**Comment by hjoliver** [comment]
No, the problem is use of Jinja2 in `global-tests.conf`. But, now that you've reminded me, I'll push up a fix.

**Comment by MetRonnie** [comment]
Closed by #5148?
